docs/sphinx/pom.xml

```diff
@@ -59,7 +59,6 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>exec-maven-plugin</artifactId>
-        <version>1.2.1</version>
```

Comment:
I'm no maven expert, so may be wrong, but I *think* by removing the version it inherits the 1.6.0 version from the root pom.  That was certainly what the `mvn versions:display-plugin-updates` output seemed to indicate.